src/test/scala/org/broadinstitute/dsde/workbench/avram/CommonTestData.scala

```diff
@@ -25,8 +32,15 @@ object CommonTestData {
     slickQueueSize = 1000
   )
 
-  val localDatabase = new SlickDatabaseFactory(localDataSourceConfig).database
```

Comment:
I liked having "local" in the name to communicate that it's talking to a local DB as opposed to Cloud SQL. I think tests should be decisive about what environment they expect to run in and the naming clue helps readers more easily understand the test.